package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@edx/edx-bootstrap": "0.4.3",
     "@edx/paragon": "1.7.1",
-    "@edx/studio-frontend": "1.2.5",
+    "@edx/studio-frontend": "1.3.2",
```

Comment:
not yet released, will fail until https://github.com/edx/studio-frontend/pull/136 lands/is published